dotcom-rendering/src/components/Card/components/ImageWrapper.tsx

```diff
@@ -168,9 +175,8 @@ export const ImageWrapper = ({
 			<>
 				{children}
 				{/* This image overlay is styled when the CardLink is hovered */}
-				{(imageType === 'picture' || imageType === 'slideshow') && (
-					<div className="image-overlay" />
-				)}
+				{(imageType === 'picture' || imageType === 'slideshow') &&
+					!hideImageOverlay && <div className="image-overlay" />}
```

Comment:
If it's an existing auto-playing slideshow the image will still dim when hovered, but not if it's a slideshow carousel. As you have to hover over the card to interact with the slideshow it would effectively mean the slideshow images are permanently dimmed as you scroll though them. I'll clarify what the desired effect is here though.